LibTest/io/Stdin/first_A01_t01.dart

```diff
@@ -27,10 +27,8 @@ run_main() async {
     await process.stderr.toList().then((errors){
       Expect.isTrue(errors.isEmpty);
     });
-    await process.stdout.toList().then((out) {
-      String res = systemEncoding.decode(out[0]);
-      // Get rid from possible new line symbols here
-      Expect.equals("Testme", res.trimRight());
+    await process.stdout.transform(systemEncoding.decoder).join().then((out) {
```

Comment:
Isn't `Stream<T>.join()` returning a  `Future<String>` ?

Then this should be rather:
```dart
final String stdout = await process.stdout.transform(systemEncoding.decoder).join();
Expect.isTrue(stdout.startsWith("Testme"));
called++:
```

Reply:
Regarding the result check:
`Expect.equals("Testme", res.trimRight());` assumes that I get exactly `Testme` string (maybe with the trimmed line ending symbols) in the stdout, and `stdout.startsWith("Testme")` does not check that I don't have any unexpected symbols at the result string. 
Going to change to `Expect.isTrue("Testme\n\n".startsWith(stdout.replaceAll("\r\n", "\n")));` fimally.

Reply:
New fix is on review